renderer/components/dashboard-projects/menu-list.tsx

```diff
@@ -19,13 +39,53 @@ export default function MenuList(props: {
           ? { top: props.state.mouseY, left: props.state.mouseX }
           : undefined
       }
+      PaperProps={{
+        style: {
+          width: 200,
+        },
+      }}
       MenuListProps={{
-        onMouseLeave: props.handleClose,
+        onMouseLeave: handleCloseMenu,
       }}
     >
-      <MenuItem onClick={props.openProjectInIde}>Show in IDE</MenuItem>
-      <MenuItem onClick={props.openProjectDirectory}>Enclosing Folder</MenuItem>
-      <MenuItem onClick={props.deleteProject}>Delete</MenuItem>
+      <MenuItem className={classes.menuItemRoot}>
+        <List className={classes.list}>
+          <ListItem onClick={openIdeExpandHandler} className={classes.listItem}>
+            <ListItemText primary="Open" />
+            {open ? <ExpandLess /> : <ExpandMore />}
+          </ListItem>
+          <Collapse in={open} timeout="auto" unmountOnExit>
+            <List className={classes.sublist}>
+              <ListItem button>
+                <div onClick={() => props.openProjectInIde('vscode')}>
+                  In Visual Studio Code
+                </div>
+              </ListItem>
+              {props.project &&
+              props.project.domain !== '' &&
+              props.project.domain !== 'angular' &&
+              props.project.domain !== 'node' ? (

```

Comment:
replace to
 
```ts
!['angular', 'node'].includes(props.project?.domain)
```
This way you avoid multiple checks. Also the "Optional chaining" helps you accessing a nested property returning undefined if any of the levels file (you could do a?.b?.c?.d?.myProperty)

Reply:
thanks